cernopendata/config.py

```diff
@@ -379,6 +379,8 @@
 FILES_REST_PERMISSION_FACTORY = allow_all
 #: Allow URI's longer than 255 chars.
 FILES_REST_FILE_URI_MAX_LEN = 512
+#: Files max size threshold(bytes)
+FILES_REST_FILE_MAX_SIZE = 200000000
```

Comment:
> `FILES_REST_FILE_URI_MAX_LEN`
> `FILES_REST_FILE_MAX_SIZE`

We should probably name the new config variable differently, so that it wouldn't seem that it *also* comes  from upstream `invenio-files-rest`... What about `CERNOPENDATA_MAX_DOWNLOAD_SIZE`?  

(Thinking of having `CERNOPENDATA_...` prefix for all variables related to our instance...)

> FILES_REST_FILE_MAX_SIZE = 200000000

Please also try to get it from environment variables; this is useful in production deployments. Similarly to:

```python
FILES_REST_FILE_URI_MAX_LEN = os.environ.get(
    "FILES_REST_FILE_URI_MAX_LEN",
    512
)
```